### DISCUSSION TITLE
Process Component | Windows 10 run as Administrator?

### DISCUSSION THREAD

**Discussion by convers39** [symfony/symfony#42658] [Q&A]
Is that possible to run a command as an administrator on Windows?

```
$process = Process::fromShellCommandline("....");
$process->??
```

**Comment by chapterjason** [comment]
You can either run the PHP process as administrator (not sure if that works) or use the [runas](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/cc771525(v=ws.11)) command.

Might be that it will not work if you have these permission prompts from windows enabled. Even then, you must be allowed to run a command for this user. But that is not symfony related.
